### PR TITLE
[#166156731] Fix opacity in loading screen of payment summary

### PR DESCRIPTION
**Pull request by laurapugliese**
parameterized opacity and increased for that screen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#166156731](https://www.pivotaltracker.com/story/show/166156731): Durante il caricamento della schermata di summary del pagamento rendere il messaggio chiaramente visibile

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by laurapugliese** [comment]
### Before: 

opacity_before

### After: 

opacity_after
opacity

File: ts/components/helpers/withLoadingSpinner.tsx [hunk 1]
```diff
@@ -19,13 +19,22 @@ const styles = StyleSheet.create({
...
+    isLoading: boolean;
+    loadingCaption?: string;
+    loadingOpacity?: number;
+  }>
 >(WrappedComponent: React.ComponentType<P>) {
   class WithLoadingSpinner extends React.Component<P> {
     public render() {
-      const { isLoading, loadingCaption } = this.props;
+      const { isLoading, loadingCaption, loadingOpacity } = this.props;
+      const opacityProps = {
```
Comment (fpersico) [inline ts/components/helpers/withLoadingSpinner.tsx]:
```suggestion
      const overlayProps = {
```

File: ts/components/helpers/withLoadingSpinner.tsx [hunk 2]
```diff
@@ -19,13 +19,22 @@ const styles = StyleSheet.create({
...
+        opacity: 1,
+        backgroundColor: `rgba(255,255,255,${loadingOpacity})`
+      };
       return (
         <Overlay
+          {...(loadingOpacity !== undefined ? opacityProps : undefined)}
```
Comment (fpersico) [inline ts/components/helpers/withLoadingSpinner.tsx]:
```suggestion
      const overlayProps = loadingOpacity !== undefined
        ? {
            opacity: 1,
            backgroundColor: `rgba(255,255,255,${loadingOpacity})`
          }
        : undefined;
      return (
        <Overlay
          {...overlayProps}
```

**Comment by fpersico** [comment]
@laurapugliese please check the comments.

File: ts/components/helpers/withLoadingSpinner.tsx (hunk 2 above)
Comment (laurapugliese) [reply ts/components/helpers/withLoadingSpinner.tsx]: Fixed in commit dec3c1e

File: ts/components/helpers/withLoadingSpinner.tsx (hunk 1 above)
Comment (laurapugliese) [reply ts/components/helpers/withLoadingSpinner.tsx]: Fixed in commit dec3c1e

**Comment by laurapugliese** [comment]
> @laurapugliese please check the comments.

Fixed all in commit dec3c1e

**Pull request merged**